Minor adjustment to the semantics of Reliable Data Transfer (Message)… (#279)
taps-api/drafts: draft-ietf-taps-interface.md

Lifetime specifies how long a particular Message can wait to be sent to the
remote endpoint before it is irrelevant and no longer needs to be
(re-)transmitted. This is a hint to the transport system -- it is not guaranteed
that a Message will not be sent when its Lifetime has expired.

> Setting a Message's Lifetime to infinite indicates that the application does

Reviewer: new paragraph?

Author: ok

not wish to apply a time constraint on it, but it does not express a need for
reliable delivery; reliability is adjustable per Message via the "Reliable Data Transfer (Message)"

> property (see {{#msg-reliable-message}}). The type and units of Lifetime are implementation-specific.

Reviewer: do this to unbreak the build:

```suggestion
property (see {{msg-reliable-message}}). The type and units of Lifetime are implementation-specific.
```

Author: Very sorry for this one!

Lifetime specifies how long a particular Message can wait to be sent to the
remote endpoint before it is irrelevant and no longer needs to be
(re-)transmitted. This is a hint to the transport system -- it is not guaranteed
that a Message will not be sent when its Lifetime has expired.

Setting a Message's Lifetime to infinite indicates that the application does

> not wish to apply a time constraint on it, but it does not express a need for

Reviewer 2: I find "a time constraint on it" confusing. Perhaps this should be "does not wish to apply a time constraint on the transmission of the message". An infinite lifetime does mean it should be transmitted at least once.

Author: I'm sorry, I merged this PR too quickly. I agree with your comment and will address it with a direct commit to the master.

Type:
: Boolean

> This property, if set to true, specifies that a message should be sent in such a way

Reviewer: tiny style nit:

```suggestion
When true, this property specifies that a message should be sent in such a way
```

Author: ack

Type:
: Boolean

When true, this property specifies that a message should be sent in such a way
that the transport protocol ensures all data is received on the other side
without corruption. Changing the ´Reliable Data Transfer´ property on Messages

> is only possible if the Connection supports reliability.

Reviewer 2: What does the 

> Connection supports reliability

 mean? The connection is using a protocol that support reliability or the application has asked for a reliable connection? If I ask for an unreliable connection and it happens to get implemented with TCP, can I ask for a reliable message?  I would say no, as this would fail most of the time. So maybe this should be "Connection is relable"?

Author: I agree with the idea of what you say, but not with the wording "Connection is reliable" because that sounds contradictory - we'd have a sentence saying that changing reliable data transfer per message is only possible if the Connection is reliable.  (but if it's reliable, why can you then change it?)

I think you mean that this should only be possible for Connections that were established with the Selection Property "Reliable Data Transfer (Connection)" enabled. So I propose to phrase it as: "Changing the ´Reliable Data Transfer´ property on Messages
is only possible for Connections that were established with the Selection Property 'Reliable Data Transfer (Connection)' enabled."

Please ACK or NACK - then I'll also apply this as a direct commit to the master. Sorry again for merging too fast.

Reviewer 2: @mwelzl -Yes that is what I  I meant, thanks.

Author: @abrunstrom: great, thanks - I now committed both these changes to the master.
